RegTesting.Tests.Framework/Properties/AssemblyInfo.cs

```diff
@@ -28,9 +28,9 @@
 //      Minor Version 
 //      Build Number
 //      Revision
-[assembly: AssemblyFileVersion("1.2.2.3")]
-[assembly: AssemblyInformationalVersion("1.2.2.3")]
+[assembly: AssemblyFileVersion("1.3.0.0")]
+[assembly: AssemblyInformationalVersion("1.3.0.0")]
 
 // Change AssemblyVersion for breaking api changes
 // http://stackoverflow.com/questions/64602/what-are-differences-between-assemblyversion-assemblyfileversion-and-assemblyin
-[assembly: AssemblyVersion("1.2.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.3.0.0")]
```

Comment:
We might not need to change the AssemblyVersion version, as your change is backwards-compatible and does not change oder remove old method signatures.
So this could be AssemblyFileVersion and AssemblyInformationalVersion = 1.2.3.0
while keeping AssemblyVersion untouched.
Older testcase projects would be fine with the new dll, while new ones could use the added method